docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -642,23 +642,46 @@ POST /example/_doc
 ===== Circle
 
 Elasticsearch supports a `circle` type, which consists of a center
-point with a radius. Note that this circle representation can only
-be indexed when using the `recursive` Prefix Tree strategy. For
-the default <<geoshape-indexing-approach>> circles should be approximated using
-a `POLYGON`.
+point with a radius.
+
+IMPORTANT: The default <<geoshape-indexing-approach,BKD tree indexing approach>>
+does not support the `circle` type. Instead, use a
+<<ingest-circle-processor,circle ingest processor>> to approximate the circle as
```

Comment:
I wonder if we should emphasise that it is not supported for indexing. A circle can actually be used in a geo_shape query?

Reply:
Good point! I'll reword to emphasize that indexing the `circle` type using the default BKD tree indexing approach is not supported.